R/html-tweak.R

```diff
@@ -129,6 +129,113 @@ tweak_footnotes <- function(html) {
   xml2::xml_remove(container)
 }
 
+# Tabsets tweaking: find Markdown recommended in https://bookdown.org/yihui/rmarkdown-cookbook/html-tabs.html
+# i.e. "## Heading {.tabset}" or "## Heading {.tabset .tabset-pills}"
+#  no matter the heading level -- the headings one level down are the tabs
+# and transform to tabsets HTML a la Bootstrap
+
+tweak_tabsets <- function(html) {
+  tabsets <- xml2::xml_find_all(html, ".//div[contains(@class, 'tabset')]")
+  if (length(tabsets) == 0) {
+    return()
+  }
+  purrr::walk(tabsets, tweak_tabset)
+  invisible(html)
+}
+
+tweak_tabset <- function(html) {
+  id <- xml2::xml_attr(html, "id")
+
+  # Users can choose pills or tabs
+  nav_class <- if (grepl("tabset-pills", xml2::xml_attr(html, "class"))) {
```

Comment:
Can you please match the style in line 201? (i.e. assignment inside if branches)